From 09b82a0004c1e7eb051b85a85680e5a8bf210a10 Mon Sep 17 00:00:00 2001
From: thanhbui <thanh.bui.fz@renesas.com>
Date: Tue, 2 Mar 2021 17:52:26 +0700
Subject: [PATCH 131/135] arm64: Kconfig : Add select SYS Clock MTU3 for
 ARCH_RENESAS

Enable MTU3 as System clock when ARCH_RENESAS enabled

Signed-off-by: thanhbui <thanh.bui.fz@renesas.com>
---
 arch/arm64/Kconfig.platforms | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/Kconfig.platforms b/arch/arm64/Kconfig.platforms
index 00ed354..a0622d0 100644
--- a/arch/arm64/Kconfig.platforms
+++ b/arch/arm64/Kconfig.platforms
@@ -186,6 +186,7 @@ config ARCH_RENESAS
 	select SOC_BUS
 	select SYS_SUPPORTS_SH_CMT
 	select SYS_SUPPORTS_SH_TMU
+	select SYS_SUPPORTS_RZG2L_MTU3
 	help
 	  This enables support for the ARMv8 based Renesas SoCs.
 
-- 
2.7.4

